Tests/Unit/LoRaTools/MacCommandTests.cs

```diff
@@ -167,35 +162,34 @@ public sealed class LinkAdrAnswerTests : MacCommandTests<LinkADRAnswer>
         { }
 
         [Fact]
-        public void FromBytes_Success() => FromBytesTest(actual => actual.Cid == Cid
-                                                                && actual.PowerAck
-                                                                && actual.DRAck
-                                                                && !actual.CHMaskAck,
-                                                         bytes => new LinkADRAnswer(new ReadOnlySpan<byte>(bytes)),
-                                                         DataBytes);
+        public void FromBytes_Success() => FromBytesTest(actual => new { actual.Cid, actual.PowerAck, actual.DRAck, actual.CHMaskAck },
+                                                         bytes => new LinkADRAnswer(new ReadOnlySpan<byte>(bytes)));
     }
 
     public sealed class LinkAdrRequestTests : MacCommandTests<LinkADRRequest>
     {
+        private static readonly Func<LinkADRRequest, object> Transform = actual => new
```

Comment:
nit: suggest to rename all `actual` to `cmd` or `command` since it doesn't necessarily represent any actual from tests (sometimes it represents the subject).

Reply:
Done in 9f2977087017f9d730427546c87734e38aa2e3ee